core/tests/protractor_utils/workflow.js

```diff
@@ -118,17 +125,24 @@ var createExplorationAsAdmin = async function() {
 // This will only work if all changes have been saved and there are no
 // outstanding warnings; run from the editor.
 var publishExploration = async function() {
-  await element(by.css('.protractor-test-publish-exploration')).isDisplayed();
-  await element(by.css('.protractor-test-publish-exploration')).click();
+  var testPublishExplorationButton = element(
+    by.css('.protractor-test-publish-exploration'));
+  await testPublishExplorationButton.isDisplayed();
```

Comment:
I think you can actually get rid of this line. `.isDisplayed()` returns whether the element is displayed, but we don't store the returned value anywhere. That means the statement doesn't actually do anything